components/nds-breadcrumbs/nds-breadcrumbs.d.ts

```diff
@@ -15,6 +16,7 @@ declare module "@nice-digital/nds-breadcrumbs" {
 		| undefined;
 
 	export interface BreadcrumbsProps {
+		[prop: string]: unknown;
```

Comment:
If we're adding `...rest` props here then should we also spread them over the wrapping element e.g.:

https://github.com/nice-digital/nice-design-system/blob/a7209e70d2a3226aed6420178e4da0013fcd6edf/components/nds-breadcrumbs/src/Breadcrumbs.js#L34

I checked and some (e.g. Tag) already spread the rest props, but some e.g. breadcrumbs don't.

Reply:
It'd be a fair amount of work to add that to every component, with tests but probably worth it if we're adding the TS defs